### ISSUE TITLE
$SomeModule fails testing because it can't find its own module

### ISSUE THREAD

**Issue by ugexe** [ugexe/zef#325]
TL:DR: try `zef nuke SiteDir` or `rm -rf ~/.zef`

`zef` has warned for awhile that distributions using `META.info` instead of `META6.json` are likely to fail installation. Unfortunately there are many distributions that still use `META.info` ( grep here: https://github.com/perl6/ecosystem/blob/master/META.list ) despite having PRs to change them, so we cannot simply remove support yet. This used to be handled by passing `-I. -Ilib` (since `-Ilib` needs no `META.json` and CURFS does not support `META.info` at all), but now we just pass `-I.` which means a different way to handle the transition was needed.

As of zef `v0.8.0` as part of the extraction phase a `META.info` file will be copied to a `META6.json`. However if the cache already contains a distribution using `META.info` it will never need to go through the extraction phase and thus never have its `META6.json` generated. To fix this one can delete the cache (`rm -rf ~/.zef` or `zef nuke SiteDir`). Disabling the cache via `--/cached` would also work, but that'd still leave the same issue next time.
